.github/workflows/run_tests.yml

```diff
@@ -145,6 +145,18 @@ jobs:
                 TF /usr/lib/redis/modules/backends/redisai_tensorflow/redisai_tensorflow.so \
                 TORCH /usr/lib/redis/modules/backends/redisai_torch/redisai_torch.so" >> $GITHUB_ENV
 
+      # Check the static build by compiling and linking examples
```

Comment:
While I know that it's "free" to run our tests on Github Actions, what do you think about doing a static build first, then doing a make clean, and then continuing on?

I don't foresee a problem with actually executing the tests once they link correctly.

Reply:
Disregard. I see this is what you're already doing

Reply:
static and shared build in separate directories, so the risk of cross-contagion is minimal -- and if it happens, we want to know about it anyway